new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java

```diff
@@ -112,5 +113,15 @@ public static boolean adminT3ChannelAccessible(String domainUID, String namespac
   public static boolean adminNodePortAccessible(String domainUID, String namespace) {
     return Domain.adminNodePortAccessible(domainUID, namespace);
   }
+  
+  /**
+   * Check if a Docker image exists.
+   * @param imageName - the name of the image to be checked
+   * @param imageTag  - the tag of the image to be checked
+   * @return true if the image does exist, false otherwise
+   */
+  public static boolean imageExists(String imageName, String imageTag) {
```

Comment:
maybe call this `doesDockerImageExist` so it is clearer what it does

Reply:
changed the method name.